lib/hooks/api/useGetWorkspaceContributors.ts

```diff
@@ -10,8 +10,24 @@ type DbWorkspaceContributor = {
   };
 };
 
-export const useGetWorkspaceContributors = ({ workspaceId, range = 30 }: { workspaceId: string; range?: number }) => {
-  const searchParams = new URLSearchParams({ range: range.toString() });
+type UseGetWorkspaceContributorsProps = {
```

Comment:
refactor (blocking): the type here is not props. Consider something like

```suggestion
type UseGetWorkspaceContributorsParams = {
````